runner/README.md

```diff
@@ -0,0 +1,57 @@
+# Runner Design #
+package runner is structs and interfaces to run Scoot processes. package runners contains implementations. This doc describes both.
+
+## Summary ##
+* Service (interface) combines Controller and StatusReader
+* Controller (interface) controls when to Run something
+  * Controller calls Invoker
+  * Controller writes to StatusWriter
+* runners.Invoker (struct) runs one command using Execer and snapshot.Filer
+  * implements Abort and Timeout
+* runners.Statuses (struct) offers read and write of RunStatus
+  * satisfies interfaces StatusWriter and StatusReader
+
+## Invoker (runner/runners/invoker.go) ##
+Invoker runs a Scoot command. This involves:
+* setup (check out a snapshot)
+* post-processing (save output)
+* babysitting (implement timeout and abort)
+* status updates while running/once finished
+
+Conceptually, it lowers a Scoot Command from a high-level abstraction (Snapshots, etc.) to an Execer command (about directories that are paths in the local filesystem).
```

Comment:
'Conceptually, it installs the command's snapshot files, then uses an Execer to run the command.  When the command has finished, it created a snapshot of the command's output.'  ?better?